bun/bun.go

```diff
@@ -21,6 +17,11 @@ import (
 	"strings"
 	"time"
 	"unicode/utf8"
+
+	"github.com/sharat87/httpbun/assets"
+	"github.com/sharat87/httpbun/exchange"
+	"github.com/sharat87/httpbun/mux"
+	"github.com/sharat87/httpbun/util"
```

Comment:
Can we move these imports back to where they were please? Thanks!

Reply:
@sharat87 This is from [goimports](https://cs.opensource.google/go/x/tools/+/master:README.md), I can move them back, however this from my understanding of go and the defacto tools library is how imports should be formatted. Here is [the commit](https://go-review.googlesource.com/c/tools/+/25145) that introduces this to the Golang org

Reply:
Hey, thanks for coming back on this quick.

1. I realize this is from `goimports`, but the current formatting is from just running `go fmt`. This is a debatable point on which one is better, but I don't see much value in changing what we already have. I'd prefer to stick with status quo here.
2. If we indeed want to change import ordering to using `goimports`, I don't think this is the PR for this. Ideally, we should do it in _all_ source files at once, so we're consistent across the codebase, and we should do it as part of a dedicated PR just for solving that problem. Not that this is something I'm inclined to doing, but just looking to keep the PR focused on just the problem being solved.

Does that sound okay?

Reply:
Your repos, your rules :+1: